addons/mail/static/src/components/message/message.js

```diff
@@ -435,16 +435,7 @@ class Message extends Component {
             ev.preventDefault();
             return;
         }
-        if (ev.target.closest('.o_mention')) {
-            this.env.messaging.openProfile({
-                id: Number(ev.target.dataset.oeId),
-                model: ev.target.dataset.oeModel,
-            });
-            // avoid following dummy href
-            ev.preventDefault();
-            return;
-        }
-        if (ev.target.closest('.o_mail_redirect')) {
+        if (ev.target.dataset.oeId && ev.target.dataset.oeModel) {
```

Comment:
Shouldn't we limit to `<a>` that have those data, just in case? Note that the test is already testing specifically with a link.

Also, not related to current PR, but shouldn't we `preventDefault` in all other cases? If the "non-link" case is used to change clicked state (as per last line of method), it shouldn't need to do any other default action such as following a potentially unparsed link either.

I think method needs to be smarter as to not change click state when clicking on non-parsed links, and following them instead.

Reply:
Ok for limiting to `<a>` with handling of dataset `oe-id` and `oe-model`.

I prefer to play safe an not change this method further. Message body contains arbitrary html content, so we should allow default browser behaviour on `<a>` with `href` in some cases. Also not entirely sure, but could have some redirects that should be handled by action manager, so any change here could likely break this.

Reply:
For for not doing more `preventDefault` but then I think when `ev.target.tagName === 'A'` it shouldn't change the message click state, because it is then not the goal of the click.